lesson_06/homeworks_06.py

```diff
@@ -21,12 +37,36 @@
 Якщо прибулець червоний, надрукуйте повідомлення про те, що гравець заробив 15 очок
 + напишіть цикл for що перебере і обробить всі значення списку alien_color
 """
+alien_color = ['green', 'red', 'yellow']
```

Comment:
```suggestion
alien_colors = ['green', 'red', 'yellow']
```
а от тут різний тип данних і треба інша змінна